test/py/ganeti.utils.log_unittest.py

```diff
@@ -141,7 +146,10 @@ def testConsole(self):
         self.assertTrue("Cannot log message" in consout)
         self.assertTrue("Test message ERROR" in consout)
 
-  class _FailingFile(file):
+    temp_file.close()
```

Comment:
The diff gets confusing but these two lines are cleanup added to `testConsole()` and then `failingFile()` gets re-declared with the revised argument.